docs/edgeql/ddl/annotations.rst

```diff
@@ -61,6 +61,96 @@ Declare an annotation ``extrainfo``.
     CREATE ABSTRACT ANNOTATION extrainfo;
 
 
+ALTER ABSTRACT ANNOTATION
+=========================
+
+:eql-statement:
+
+
+Change the definition of an :ref:`annotation <ref_datamodel_annotations>`.
+
+.. eql:synopsis::
+
+    ALTER ABSTRACT ANNOTATION <name>
+    [ "{" ] <subcommand>; [...] [ "}" ];
+
+    # where <subcommand> is one of
+
+      RENAME TO <newname>
+      EXTENDING ...
```

Comment:
Abstract annotations don't inherit, so remove this.

Reply:
OK, I'll remove that from docs and grammar.